src/components/UI/TodoTable.tsx

```diff
@@ -6,20 +6,29 @@ import {
   TableCell,
   TableBody,
   Paper,
+  Alert,
+  LinearProgress,
 } from "@mui/material";
+import { useMutation, useQuery } from "react-query";
+
+import { deleteTodo } from "../../services/queries/delete-todo";
+import Todo from "../../models/todo";
+
+const TodoTable: ({ todos }: { todos: Todo[] }) => JSX.Element = ({
+  todos,
+}) => {
+  const { mutate } = useMutation(deleteTodo);
+
+  const fetchTodo = () => {
+    return todos;
+  };
+
+  const { data, isLoading, isError } = useQuery("todos", fetchTodo);
```

Comment:
This seems very weird. We are always just returning todos that are passed in as props? What's the point of using `useQuery` in here?